### PR TITLE
Fix hinge loss + add gan documentation

### PR DESCRIPTION
**Pull request by EmanueleGhelfi**

### CODE REVIEW

File: ashpy/keras/losses.py [hunk 1]
```diff
@@ -263,6 +263,5 @@ def reduction(self, value: tf.keras.losses.Reduction) -> None:
 
     def call(self, d_real: tf.Tensor, d_fake: tf.Tensor) -> tf.Tensor:
         """Computes the hinge loss"""
```
Comment (galeone) [inline ashpy/keras/losses.py:302]: @EmanueleGhelfi can you update the documentation of the `__init__` method of this class, explaining **why** the hinge-loss is computed in this way for the generator?

Comment (galeone) [reply ashpy/keras/losses.py:302]: Or better, the class documentation and not the `__init__` doc

**Comment by galeone** [comment]
Does this PR close #23?

**Comment by EmanueleGhelfi** [comment]
This PR closes #23 and #22

**Review by galeone — APPROVED** [review]
